clients/fossid-webapp/src/test/kotlin/FossIdClientReturnTypeTest.kt

```diff
@@ -256,6 +257,28 @@ class FossIdClientReturnTypeTest : StringSpec({
         }
     }
 
+    "When create scan returns an error, no exception is thrown" {
+        service.createScan(
+            "",
+            "",
+            PROJECT_CODE_1,
+            SCAN_CODE_1,
+            "git_repo_url",
+            "develop"
+        ).shouldNotBeNull().run {
```

Comment:
Nit: Can this use the same `shouldNotBeNull` variant as in other test cases that does not require the `run` ?